lib/zipkin-tracer/faraday/zipkin-tracer.rb

```diff
@@ -17,6 +17,7 @@ def call(env)
         b3_headers.each do |method, header|
           env[:request_headers][header] = trace_id.send(method).to_s
         end
+        env[:request_headers]['X-B3-Sampled'] = env[:request_headers]['X-B3-Sampled'] == 'true' ? '1' : '0'
```

Comment:
At that point there would it not be simpler to adjust the inbound test  to be something like
(lib/zipkin-tracer/rack/zipkin-tracer.rb)
`['1','true'].include? parent_trace_sampled`
That would allow the flexibility of the field on the inbound to accept '1'/'true' for compatibility and force '1' on the outbound?  The edge translation between http and the TraceId object is kind of messy.  Perhaps a dedicated object would be a better abstraction?  Something like
class TraceIdHttpHeaderConverter
  def toTraceId
  def toHeaders
end
Naming conventions could probably be a little better but you get the gist.  Let me know what you think.